ion_schema_2_0/constraints/byte_length.isl

```diff
@@ -74,16 +74,24 @@ $test::{
     { byte_length: range::(1 2) },
     { byte_length: range::[min, max] },
     { byte_length: range::null.list },
-    { byte_length: range::[2, 1] },
     { byte_length: range::[1] },
     { byte_length: range::[1, 2, 3] },
     { byte_length: range::[1d0, 2] },
     { byte_length: range::[1, 2d0] },
     { byte_length: range::[1e0, 2] },
     { byte_length: range::[1, 2e0] },
+  ]
+}
+
+$test::{
+  description: "byte_length range must not be non-empty",
```

Comment:
The phrase "must not be non-empty" occurs in many of these, and it really confused me until I realized that "empty" referred to the values that could satisfy the range, rather than the range declaration itself. But even if "empty" weren't confusing, don't we mean "must not be empty"?

Reply:
You're right—there's an accidental double negative there.

"range must be satisfiable" maybe?